.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!--
+  This template is not mandatory.
+  It simply serves to provide a guide to allow a better review of pull requests.
+-->
+### What's is in this PR ?
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+
+- [ ] Change that requires a documentation update
+- [ ] Change that adds new dependencies
+
+### What's this PR do?
+
+<!--
+  Please include a summary of the change and which issue is fixed.
+  Please also include relevant motivation and context.
+  List any dependencies that are required for this change.
+-->
+
+### What are the relevant tickets?
+
+<!-- Link issues here -->
+
+### Where should the reviewer start?
+
+<!-- Tell which files are relevant or which parts of the code you are least confident about -->
+<!-- You can link a entire file of you branch like this : https://github.com/<owner>/<project>/blob/<branch-name>/<path-to-the-file> -->
+
+### How should this be manually tested?
```

Comment:
I've mixed feelings about that one. I would _personally_ prefer to see that part gone, but I won't battle over it if it stays.

Pro: sometimes it's interesting to help reviewers by describing how they can view the changes by themselves

Con: useless in most cases
* Fixed issues must include unit and/or functional tests, validating the fix (i.e. tests reproducing the problem and failing without the fix)
* New features must include new functional tests
* New features may be complex and thus difficult to detail in that section

Reply:
I suggest to maintain this section as an optional one (each PR submitter is free to keep it or not, depending of its pertinence in a specific case)

Reply:
fine by me